src/darwin/Framework/CHIPTests/TestHelpers/MTRTestServerAppRunner.h

```diff
@@ -16,21 +16,14 @@
 
 #import <Foundation/Foundation.h>
 
-@class MTRTestCase;
+#import "MTRTestCase.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
-/**
- * A representation of a server application instance.
- *
- * Server applications are assumed to be compiled into out/debug/${APPNAME}-app,
- * with the binary being out/debug/${APPNAME}-app/chip-${APPNAME}-app.
- */
-@interface MTRTestServerAppRunner : NSObject
+@interface MTRTestCase (ServerAppRunner)
```

Comment:
Should probably rename this file to `MTRTestCase+ServerAppRunner.h` now

Reply:
Yep, done.